src/Utils/Utils.php

```diff
@@ -199,21 +199,21 @@ public static function mapKeyValue($traversable, callable $fn)
     }
 
     /**
-     * @param mixed|Traversable $traversable
+     * @param iterable<mixed> $iterable
```

Comment:
I'd also add a documentation for key everywhere (`iterable<keytype, valuetype>`)

Reply:
Feel free to add a PR.